components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -375,4 +378,18 @@ void SecureZeroData(void* data, size_t size) {
 #endif
 }
 
+// Updates preferences for when the wallet is unlocked.
+// This is done in a utils function instead of in the KeyringController
+// because we call it both from the old extension and the new wallet when
+// it unlocks.
+void UpdateLastUnlockPref(PrefService* prefs) {
+  base::Time time = base::Time::Now();
+  base::Time::Exploded exploded;
+  time.LocalExplode(&exploded);
+  std::string ymd = base::StringPrintf("%d-%02d-%02d", exploded.year,
+                                       exploded.month, exploded.day_of_month);
+  prefs->SetString(kBraveWalletLastUnlockYMD, ymd);
```

Comment:
if we make it a time preference, this function can be shorted to a single line: `prefs->SetTime(kReferralTimestamp, base::Time::Now());`

Reply:
not the right pref name, but I see the code you grep'ed from 👍

Reply:
done